etc/kube/start-minikube-vm.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -x
+
+cd ${GOPATH}/src/github.com/pachyderm/pachyderm
+
+# push_to_minikube pushes dockers images to the minikube vm
+function push_to_minikube {
```

Comment:
This seems like it would be useful as a script, right? Maybe split this out?

Reply:
Done. This is now in `etc/kube/push-to-minikube.sh`